bin/generate.py

```diff
@@ -1320,7 +1346,9 @@ def parse(key, name, yaml, parent):
                                     f'Unnumbered testcases must not have an empty key: {Path("data") / d.path / child_name}/\'\''
                                 )
                         c = parse(child_key, child_name, child_yaml, d)
-                        if c is not None:
+                        if isinstance(c, Sequence):
```

Comment:
where does this come from?

Reply:
a single entry can now return multiple rules i.e. if count > 1

Reply:
Ahh right; can you add that as a comment at the top of the `parse` function?

But `ts = []; ...; return ts` should just return a `list` right? No need to import `Sequence`?